### PR TITLE
feat: make duplicates' helpers supporting combo assays as well

### PR DESCRIPTION
**Pull request by gladkia**
See: https://code.roche.com/gdr/gdrviz/-/merge_requests/147

### CODE REVIEW

**Review by bczech — APPROVED** [review]

File: tests/testthat/test-duplicates.R [hunk 1]
```diff
@@ -52,21 +52,40 @@ test_that("get_duplicated_rows works as expected", {
 })
 
 test_that("[has|get]_assay_dt_duplicated_rows works as expected", {
-  
+ 
+  # single-agent data
   sdata <- get_synthetic_data("finalMAE_small")
   smetrics_data <- convert_se_assay_to_dt(sdata[[1]], "Metrics")
```
Comment (j-smola) [inline tests/testthat/test-duplicates.R:58]:
```suggestion
  smetrics_data <- convert_se_assay_to_dt(sdata[[get_supported_experiments("sa")]], "Metrics")
```

File: tests/testthat/test-duplicates.R [hunk 2]
```diff
@@ -52,21 +52,40 @@ test_that("get_duplicated_rows works as expected", {
...
+  expect_true(nrow(empty_dt) == 0)
+  expect_is(empty_dt, "data.table")
   
-  expect_equal(get_assay_dt_duplicated_rows(smetrics_data), 1:200)
-  expect_equal(dim(smetrics_data), dim(get_assay_dt_duplicated_rows(smetrics_data, output = "data")))
-  expect_equal(get_assay_dt_duplicated_rows(smetrics_data_f), integer(0))
-  empty_dt <- get_assay_dt_duplicated_rows(smetrics_data_f, output = "data")
+  # combo data
+  cdata <- get_synthetic_data("finalMAE_combo_matrix_small")
+  cscores_data <- convert_se_assay_to_dt(cdata[[1]], "scores")
```
Comment (j-smola) [inline tests/testthat/test-duplicates.R:76]:
```suggestion
  cscores_data <- convert_se_assay_to_dt(cdata[[get_supported_experiments("combo")]], "scores")
```

File: tests/testthat/test-duplicates.R [hunk 3]
```diff
@@ -75,14 +94,9 @@ test_that("throw_msg_if_duplicates works as expected", {
  
   sdata <- get_synthetic_data("finalMAE_small")
   smetrics_data <- convert_se_assay_to_dt(sdata[[1]], "Metrics")
```
Comment (j-smola) [inline tests/testthat/test-duplicates.R:96]:
```suggestion
  smetrics_data <- convert_se_assay_to_dt(sdata[[get_supported_experiments("sa")]], "Metrics")
```

**Review by j-smola — APPROVED** [review]

File: tests/testthat/test-duplicates.R (hunk 1 above)
Comment (j-smola) [reply tests/testthat/test-duplicates.R]: ✔️

File: tests/testthat/test-duplicates.R (hunk 2 above)
Comment (j-smola) [reply tests/testthat/test-duplicates.R]: ✔️

File: tests/testthat/test-duplicates.R (hunk 3 above)
Comment (j-smola) [reply tests/testthat/test-duplicates.R]: ✔️

**Pull request merged**